docs-kits/kits/Industry Core Kit/Software Development View/page_digital-twins.mdx

```diff
@@ -241,50 +247,186 @@ If a data provider no longer creates assets on the level of submodels, the Conne
+
+#### Data Consumption with AAS Submodel Descriptor to represent mass data (like parquet files) reference
 
 A data consumer must first identify the correct submodel descriptor for a Catena-X data transfer. For Catena-X submodel descriptors, the `subprotocol` attribute must have the value "DSP". Then, the data consumer must use the information in the `subprotocolBody` to perform a contract negotiation for the asset referenced by `id` with the Connector of the data provider specified by `dspEndpoint` and afterwards initiate the data transfer.
 
 Further information about the data transfer of submodels can be found in the [Digital Twin KIT](../../../../../docs-kits/category/digital-twin-kit).
 
-### Lookup in the Digital Twin Registry
+**Cloud2Cloud File transfer with EDC Cloud extension**
+
+1. `Provider` prepares aasx file with parquet file.
+2. `Provider` uploads the aasx file to the cloud (S3 bucket in our case)
+3. `Provider` creates an edc-asset. The dataAddress includes information about the S3 bucket and the file name:
+
+```json
+ {
+   "edc:dataAddress": {
+     "edc-type": "Cloud type e.g. AmazonS3",
+     "edc:bucket": "<provider-bucket>",
+     "edc:keyName": "<file-name>"
+   }
+ }
+```
+
+See also full example here: [edc-asset for file transfer](https://eclipse-tractusx.github.io/docs-kits/kits/Quality-Kit/Software%20Development%20View/Specification%20Quality%20Kit/)
+4. `Provider` creates a shell with shell descriptors and the edc Asset with controlplane URL.
+5. `Consumer` calls the DTR via EDC and read the submodeldescriptor from the shell.
+6. `Consumer` starts negotiation/transfer process and provide their own bucket information:
+
+```json
+ {
+   "edc:dataDestination": {
+     "edc-type": "Cloud type e.g. AmazonS3",
+     "edc:bucket": "<consumer-bucket>",
+     "edc:keyName": "<file-name>",
+     "edc:accessKeyId":"<access-key-id>",
+     "edc:secretAccessKey":"<secretAccessKey>"
+   }
+ }
+```
+
+7. `Provider` retrieve the request and download the requested aasx file from bucket and upload it to the consumer bucket.
+8. `Consumer` can download the file from their own bucket.
+
+**SubmodelDescriptor for aasx  file (Parquet Data - via File Transfer)**
+
+> ⚠ **Hint:**
+> Since the Cloud2Cloud file transfer is an asynchronous call and the consumer only triggers the process via EDC, the workflow cannot currently be fully represented in the submodel descriptors.
+> An issue has already been created with the IDTA to extend the submodel descriptor so that an asynchronous call can be defined as a standard feature. See [github-issue 347](https://github.com/admin-shell-io/aas-specs-api/issues/347)
+
+For the file transfer via S3 bucket the submodelDescriptor can be defined like:
+
+
+```json
+{
+  "submodelDescriptors": [
+    {
+      "idShort": "quality-data",
```

Comment:
```suggestion
      "idShort": "qualityData",
```

Reply:
In V3.0 of AAS Spec no hyphens in idShort are allowed. With V3.1 hypens would be allowed. Since not relevant for example use idShort without Hyphen.